journal/src/main/scala/com/evolutiongaming/kafka/journal/Journal.scala

```diff
@@ -329,4 +330,34 @@ object Journal {
 
     implicit val configReaderCallTimeThresholds: ConfigReader[CallTimeThresholds] = deriveReader[CallTimeThresholds]
   }
+
+  sealed trait ConsumerPoolSize {
+
+    def calculate[F[_]: Runtime: Applicative]: F[Int] =
+      this match {
+        case ConsumerPoolSize.Fixed(size) =>
```

Comment:
nice idea, but… do we ever need this?

Reply:
Removed "fixed" variant, now we only have "scale with CPUs" variant.